packages/components/pie-link/test/visual/pie-link.spec.ts

```diff
@@ -52,3 +54,25 @@ componentVariants.forEach((variant) => test(`Render all prop variations for Vari
 
     await percySnapshot(page, `PIE Link - Variant: ${variant}`);
 }));
+
+const icon = '<svg slot="icon" xmlns="http://www.w3.org/2000/svg" role="presentation" focusable="false" fill="currentColor" viewBox="0 0 16 16" class="c-pieIcon c-pieIcon--plusCircle"><path d="M8.656 4.596H7.344v2.748H4.596v1.312h2.748v2.748h1.312V8.656h2.748V7.344H8.656V4.596Z"></path><path d="M12.795 3.205a6.781 6.781 0 1 0 0 9.625 6.79 6.79 0 0 0 0-9.625Zm-.927 8.662a5.469 5.469 0 1 1-7.734-7.735 5.469 5.469 0 0 1 7.734 7.736Z"></path></svg>';
+
+iconPlacements.forEach((iconPlacement) => test(`Render icon slot for iconPlacement: ${iconPlacement}`, async ({ page, mount }) => {
```

Comment:
Should we combine these into a single screenshot?

Reply:
I think it makes sense to combine the leading/trailing states into one

Reply:
good one, updated!